lib/mayaUsd/resources/scripts/CMakeLists.txt

```diff
@@ -9,6 +9,13 @@ list(APPEND scripts_src
     mayaUsdMayaReferenceUtils.py
 )
 
+list(APPEND maya_lib_scripts_src
```

Comment:
USD cache creation options moved out of UI code to separate Python module, under mayaUsd.lib